modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1643,6 +1643,10 @@
 			<category name="Upgrade" />
 			<description name="Replaces the references of the method `HtmlUtil.extractText(` with the method `extractText(` of `HtmlParser` class" />
 		</check>
+		<check name="UpgradeJavaFacetedSearcherCheck">
+			<category name="Upgrade" />
+			<description name="Replaces the references of the `Indexer indexer = FacetedSearcher.getInstance();` declaration and `indexer.search(` method call." />
```

Comment:
@nicolas-sss, can we use `indexer.search()` or `indexer.search` instead?

It seems a little strange we document it using just a single parenthesis